index.html

```diff
@@ -276,22 +276,22 @@ <h5 class="subtitle">We are always in touch we the outside world</h5>
       </div>
     </div>
     <div class="tabs is-boxed is-centered main-menu" id="nav">
-      <ul>
-        <li data-target="pane-1" id="1" class="is-active" tabindex="0" role="none">
-          <a role="menuitem">
-            <span class="icon is-small"><i class="fa fa-image"></i></span>
+      <ul role="tablist">
+        <li data-target="pane-1" id="1" class="is-active" tabindex="0" role="tab">
```

Comment:
Please add `aria-controls=""` for neaded **<li**

<img width="456" alt="image" src="https://user-images.githubusercontent.com/16785774/172826352-d3df3aec-fc8c-4627-b289-d1cd91afe9c4.png">